test/tests/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+yum install nodejs npm
+npm -g install phantomjs
+
+yum install yum-plugin-copr
+yum copr enable lmr/Autotest
+yum install avocado
+yum install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+setenforce 0
+
+mkdir -p /usr/share/avocado/tests/
+
+/bin/cp -rf  * /usr/share/avocado/tests/; python checklogin.py ; cat /root/avocado/job-results/latest/job.log 
```

Comment:
I'm glad that this works now.
there will be easy way how to deploy based on your current workflow based on using VMs:
1) using mentioned virt-deploy (or you can create some default snap manually) or virt images based on some kickstart installation via virt-install
2) use avocado vms runner (http://avocado-framework.readthedocs.org/en/latest/VirtualMachinePlugin.html) like: # avocado run --vm-domain fedora20 --vm-hostname 192.168.122.30 --vm-username autotest --vm examples/tests/sleeptest.py examples/tests/failtest.py
unfortunatelly this part of avocado framework is broken now, because of bug. (I've discussed  it yesterday with developer and he promised to repair it ASAP. So that I will be able to provide fully working prototype also with virt environment setup&run.